### ISSUE TITLE
Feature request: a way to identify where a kit is being used

### ISSUE THREAD

**Issue by renaissance-design** [FortAwesome/Font-Awesome#19907]
### Is your feature request related to a problem?

I just got an email this morning saying that a kit that I apparently created 4 years ago has used up 60% of its allowed pageviews, so I logged in to figure out what Font Awesome is (it's clearly so awesome that it stuck indelibly in my memory) find that the kit is named an impenetrable string, the pageview counter has zero information on where the requests are coming from, so it essentially amounts to "trust us bro, you're soon going to owe us money or something we refuse to identify will stop working".

### Feature description

A pageview tracker that identifies where these "pageviews" are allegedly occurring. 

### Additional context

I dislike shady scammers, and you're walking and quacking exactly like shady scammers.

### Feature request checklist

- [X] The title starts with "Feature request: " and is followed by the requested feature description
- [X] This is a single feature
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by sensibleworld** [comment]
Yup. Hoping to add this information to the UI soon. In the meantime, if you reach out to `hello@fontawesome.com`, we can do a reverse lookup for you.

**Comment by radzisz** [comment]
More than a half year passed - FontAwesame continues its scamming mailing, and there is still no promised information in the UI - so sad -
